lib/ldp/resource/rdf_source.rb

```diff
@@ -17,7 +17,9 @@ def initialize client, subject, graph_or_response = nil, base_path = ''
 
     def create
       super do |req|
-        req.headers = { "Content-Type" => "text/turtle" }
+        if req.headers["Content-Type"] != "text/turtle"
```

Comment:
so just remove the if-conditional and just set:

req.headers["Content-Type"] = "text/turtle"

right? can't remember why I made the conditional :)